examples/cloud/nativeModelLookupConsole.php

```diff
@@ -39,88 +39,11 @@
  * - 51degrees/fiftyone.devicedetection
  */
 
-require_once(__DIR__ . "/exampleUtils.php");
 require_once(__DIR__ . "/../../vendor/autoload.php");
```

Comment:
do we need this require here?

Reply:
Yes, otherwise running the examples according to the instructions in the readme will result in a `Class ... not found` error.